R/modlib.R

```diff
@@ -55,7 +55,8 @@
 ##' mod <- mread("viral2", modlib())
 ##' mod <- mread("pred1",  modlib())
 ##' mod <- mread("pbpk",   modlib())
-##' mod <- mread("1005",   modlib()) # embedded NONMEM result
+##' mod <- mread("1005",   modlib())      # embedded NONMEM result
+##' mod <- mread("nm-like", moodlib()) # model with nonmem-like syntax
```

Comment:
typo: `moodlib()`

Reply:
Thanks. Fixed.

Reply:
I also pushed a change to the Makefile to get the modlib build tests "un-skipped" (it wasn't doing that); don't think this would have pick up the typo in the example, but fwiw.